### DISCUSSION TITLE
FCU Vertical Speed offset

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#4855] [Help]
With the DEV mode the FCU settings of Vertical speed are no longer in the standard offset 07F2 (while speed is still in 07E2). 
Whre can I find it now?

**Comment by tracernz** [comment]
Docs are kept up-to-date:
https://github.com/flybywiresim/a32nx/blob/master/docs/a320-events.md
https://github.com/flybywiresim/a32nx/blob/master/docs/a320-simvars.md

Not really sure what the "offset" relates to. Maybe specific to the SimConnect client you're using.

**Reply by MStallen** [reply]
They are FSUIPC offsets. Still looking if FBW is using offsets and or lvars for the value of the choosen vertical speed.

**Reply by aguther** [reply]
You find the answer in the docs, it’s local variables in the case of the V/S or FPA. Furthermore a good overview for the FCU is here:
https://docs.flybywiresim.com/start/autopilot-fbw/
